### ISSUE TITLE
[question] Training/validation/test splits

### ISSUE THREAD

**Issue by oarriaga** [kimhc6028/pathnet-pytorch#3]
Hello again!
I was looking at your `mnist_dataset.py` file and I was wondering if you create any validation/test splits for this task. I am asking since I am trying to figure out if the `fitness/loss` value is calculated using the training data itself or if you calculate it using a validation/test split.
And in case you create this splits, are you adding the salt and pepper noise also to them?
Thank you!
 
Maybe @jaesik817 can comment on this issue as well.

**Comment by oarriaga** [comment]
I just found the answer in the paper:
` The fitness of that pathway is the proportion of correct examples classified on the training set during this training period.`
Thank you
